code/datums/gamemode/factions/bloodcult/bloodcult_flavourlines.dm

```diff
@@ -0,0 +1,356 @@
+		"Your closed mind dishonours you." = 3,
+		"Our secrets were beyond your understanding." = 3,
+		"My science was not for wimps such as you." = 3,
+	),
+	CIVILIAN_POSITIONS = list(
+		"A little job in life, and a forgotten death." = 3,
+		"This refusal is but a footnote on my story." = 3,
+		"You refused the only opportunity you had to make a difference." = 3,
+	),
+	CARGO_POSITION = list(
+		"I care little for the hoarders of your kind." = 3,
+		"As expected for a glorified crate handler." = 3,
+	),
+	SECURITY_POSITION = list(
+		"You already failed." = 3,
+		"This refusal does not erase your failure." = 3,
+		"Your stubbornness amuses me. I already won." = 3,
+		"I could have freed you." = 3,
+		"You will always remain on the weaker side." = 3,
+	),
+)
+
+var/list/acceptance_lines_by_dept = list(
+	COMMAND_POSITIONS = list(
+		"I knew you had it in you." = 3,
+		"The chains of commanding are broken." = 3,
+		"Be ready to lead the stronger side." = 3,
+		"Arise, new champion." = 3,
+	),
+	ENGINEERING_POSITIONS = list(
+		"The forges of the Geometer welcome you." = 3,
+		"Your true potential has been unraveled. " = 3,
+		"Forge the sword that will slay my enemies." = 3,
+		"Arise, new craftsman." = 3,
+	),
+	MEDICAL_POSITIONS = list(
+		"The blood was always your companion." = 3,
+		"You healed so many... but only now are you truly alive." = 3,
+		"Arise, new healer." = 3,
+	),
+	SCIENCE_POSITIONS = list(
+		"This always was your calling." = 3,
+		"The secrets of the veil are now yours to research." = 3,
+		"The logical conclusion to your career choice." = 3,
+		"Was it not what you always wanted?" = 3,
+		"Arise, new adept." = 3,
+	),
+	CIVILIAN_POSITIONS = list(
+		"Only here will you be fullfilled." = 3,
+		"A task has finally been given to you." = 3,
+		"Rise up." = 3,
+		"And there goes a life of servitude." = 3,
+		"Arise, new peon." = 3,
+	),
+	CARGO_POSITIONS = list(
+		"When this is over, expect much more than your dreamed 'Cargonia'." = 3,
+		"Be the hand that arms my soldiers." = 3,
+		// Need to find an "arise" line
+	),
+	SECURITY_POSITIONS = list(
+		"Congratulations on joining the stronger side." = 3,
+		"The corporate slave died. Let a new, free man take their place." = 3,
+		"You have finally seen the light." = 3,
+		"Your freedom begins at this hour, in this place." = 3,
+		"Arise, new warrior." = 3,
+	),
+)
+
+var/list/failure_lines_by_specific_job = list(
+	"Paramedic" = list(
+		"You will not save anyone from where I sent you." = 5,
+	),
+	"Trader" = list(
+		"I offered you a home, and you refused." = 5,
+	),
+	"Captain" = list(
+		"Do you feel in charge?" = 5,
+	),
+)
+
+var/list/acceptance_lines_by_specific_job = list(
+	"Trader" = list(
+		"And here ends your wandering." = 5,
+		"No longer will you be content with scraps." = 5,
+	),
+)
+
+var/list/failure_lines_by_specific_race = list(
+	/datum/species/plasmaman = list(
+		"Your loyalty to this company that made you a living dead is amusing." = 3,
```

Comment:
Feels a bit awkward. Maybe "Your loyalty to the company that twisted you into the living dead is amusing."?